php/api/controller/class-audit-posts-controller.php

```diff
@@ -139,13 +141,36 @@ public function register_routes() {
 			'schema' => array( $this, 'get_public_item_schema' ),
 		) );
 
-		register_rest_route( $this->namespace, '/' . $this->rest_base . '/' . static::PROJECT_PATTERN . '/' . static::SLUG_PATTERN, array(
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/' . static::PROJECT_PATTERN . '/' . static::PROJECT_TYPE_PATTERN, array(
 			'args'   => array(
-				'project' => array(
+				'project'      => array(
```

Comment:
@rheinardkorf `project` alone seem ambiguous, the description of `The users slug` makes it very confusing. This is not specific to this PR, just a general question since we're messing with the schema it might make sense to make any other changes at this time.

Reply:
@valendesigns what about `project_client`?  The confusion is because "at this moment" it actually relates to a user's login name.  As needs change it may not be... what would be a sensible identifier?